astropy/table/table.py

```diff
@@ -2011,48 +2011,44 @@ def _convert_string_dtype(self, in_kind, out_kind, python3_only):
 
         self._init_from_cols(newcols)
 
-    def convert_bytestring_to_unicode(self, python3_only=False):
+    def convert_bytestring_to_unicode(self, **kwargs):
```

Comment:
Probably worth thinking about using an approach like I presented in https://github.com/astropy/astropy/pull/6655/files#r143031672 here as well.